### PR TITLE
test: add new test cases CliKintoneTest-62-63-64

### PR DESCRIPTION
**Pull request by hung-cybo**
## Why

Adding e2e test cases.

## What

- adding test cases to verify --encoding option
  - utf8 encoding
  - sjis encoding
  - unsupported character code

## How to test

```
pnpm build
pnpm test:e2e
```

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/cli-kintone/blob/main/CONTRIBUTING.md)
- [ ] Updated documentation if it is required.
- [x] Added/updated tests if it is required. (or tested manually)
- [x] Passed `pnpm lint` and `pnpm test` on the root directory.

### CODE REVIEW

File: features/step_definitions/import.ts [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
 Given(
   "The csv file {string} with content as below:",
   async function (filePath: string, table) {
-    await this.generateCsvFile(table.raw(), filePath);
+    await this.generateCsvFile(table.raw(), { filePath });
+  },
+);
+
+Given(
+  "The csv file {string} with {string} content as below:",
```
Comment (tasshi-me) [inline features/step_definitions/import.ts]: It's better to describe what users have to specify on `{string}`

```suggestion
  "The csv file {string} with {string} encoded content as below:",
```

**Review by tasshi-me — COMMENTED** [review]
I added one comment about a step definition.
Please give me your opinion.

File: features/step_definitions/import.ts (hunk 1 above)
Comment (hung-cybo) [reply features/step_definitions/import.ts]: Yes. I think the new step definition is better.
It makes the step more clear.

**Review by tasshi-me — APPROVED** [review]
LGTM!

**Pull request merged**